<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>397</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>397</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScaleUp</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="sourceLabel">
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="FileSelection" name="sourceFileSelection"/>
   </item>
   <item>
    <widget class="QLabel" name="destinationLabel">
     <property name="text">
      <string>Destination:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="FileSelection" name="destinationFileSelection">
     <property name="existingFile">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="codecLayout">
     <item>
      <layout class="QVBoxLayout" name="videoCodecLayout">
       <item>
        <widget class="QLabel" name="videoCodeLabel">
         <property name="text">
          <string>Video codec:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="videoCodecInput"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="audioCodecLayout">
       <item>
        <widget class="QLabel" name="audioCodecLabel">
         <property name="text">
          <string>Audio codec:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="audioCodecInput"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="upscalerModelLabel">
     <property name="text">
      <string>RealESGRAN model:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="upscalerModelInput"/>
   </item>
   <item>
    <widget class="QLabel" name="scaleLabel">
     <property name="text">
      <string>Scale factor:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="scaleInput">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelection</class>
   <extends>QFrame</extends>
   <header>ScaleUp.gui.FileSelection.FileSelection</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
